helm/trivy/templates/kyverno-policy-exception.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.kyvernoPolicyExceptions.enabled }}
+  {{- if .Capabilities.APIVersions.Has "kyverno.io/v2alpha1/PolicyException" -}}
+apiVersion: kyverno.io/v2alpha1
+kind: PolicyException
+metadata:
+  name: {{ include "trivy.fullname" . }}-exception
+  {{- if .Values.kyvernoPolicyExceptions.namespace }}
+  namespace: {{ .Values.kyvernoPolicyExceptions.namespace }}
+  {{- else }}
+  namespace: {{ .Release.Namespace }}
+  {{- end }}
+spec:
+  exceptions:
+  - policyName: disallow-capabilities-strict
+    ruleNames:
+    - require-drop-all
+    - autogen-require-drop-all
+  - policyName: restrict-seccomp-strict
+    ruleNames:
+    - check-seccomp-strict
+    - autogen-check-seccomp-strict
+  match:
+    any:
+    - resources:
+        kinds:
+        - StatefulSet
+        - Pod
+        namespaces:
+        - {{ .Release.Namespace }}
+        names:
+        - {{ include "trivy.fullname" . }}
+        - {{ include "trivy.fullname" . }}-*
```

Comment:
I think this could be handled with just one name, right?

Reply:
Just wanted it to be more specific/exclusive but yes, it will work with one name.